### DISCUSSION TITLE
[Features]Features Suggestion

### DISCUSSION THREAD

**Discussion by leo719044** [YetiForceCompany/YetiForceCRM#15056] [Wishlist]
Thanks to Yetiforce team for the great job have done. 

I am currently searching for the CRM for my team. Frankly I was hestitate when choosing Yetiforce or Vtiger. I was deeply impressed by the fashion UI design of Yetiforce. and some features developed by the team. 

Also I noticed there are some features are not ready in the latest version of Yetiforce. So I take a note here for remember. 
1. Attachment upload in comment section.
![image](https://user-images.githubusercontent.com/88583995/128954991-36d2a3ed-6fd3-4b96-87a6-9c30d31aeea9.png)

2. Attach Tag to ACCOUNT/ TICKET etc. 
![image](https://user-images.githubusercontent.com/88583995/128954944-bee4d4dd-8f5a-447d-8715-db0f0b02f9cb.png)

3. Generate report 
![image](https://user-images.githubusercontent.com/88583995/128954884-73ba7aca-5185-45d5-a8b0-16b8bc17b06f.png)

**Comment by JEEVAN28** [comment]
I also find Add Tag, Comment Attachment and Report should be added as feature, I only found these 3 things missing after migrated from Vtiger to Yetiforce
